scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -129,29 +110,48 @@
       </div>
     </div>
     {% endif %}
-    {% if case.cnv_report %}
-      {{ cnv_report(institute._id, case) }}
-    {% endif %}
-    {% if case.gene_fusion_report or case.gene_fusion_report_research %}
-      {{ gene_fusion_reports(institute._id, case) }}
-    {% endif %}
-    {% if case.coverage_qc_report %}
-      {{ coverage_qc_report(institute._id, case) }}
-    {% endif %}
-    {% if case.delivery_report %}
-      {{ delivery_report(institute._id, case) }}
-    {% endif %}
+
+    <!-- Custom case reports as in constants.CUSTOM_CASE_REPORTS -->
+    {% for _, report_type in report_types.items() %}
+      {{ custom_case_report(institute, case, report_type) }}
+    {% endfor %}
+
+    <!-- Delivery reports for old analyses -->
     {% if case.analyses %}
       {% for analysis in case.analyses %}
         {% if analysis.delivery_report %}
-          {{ analysis_report( institute._id, case, analysis) }}
+          {{ analysis_report( institute._id, case, analysis ) }}
         {% endif %}
       {% endfor %}
     {% endif %}
 </div>
 <!-- end reports macro -->
 {% endmacro %}
 
+{% macro custom_case_report(institute, case, report_type) %}
+  {% if case["{}".format(report_type)] %}
```

Comment:
😸

Reply:
Had to do it in this ugly way. Jinja has no case $report_type or something similar..

Reply:
Huh? I would have naively thought
```suggestion
  {% if case[report_type] %}
```
or better
```suggestion
  {% if case.get(report_type) %}
```
would just work? 😊

Reply:
We learn each day!

Reply:
Wait you are right, it is actually working! I don't know what I wrote that didn't work yesterday, but I'll fix this way: it's way easier! 😆

Reply:
Being naive pays off in simplicity! 😄